articles.tex

```diff
@@ -253,7 +253,7 @@
 
 \asection{Advisory Membership}
 \asubsection{Advisory Membership Qualifications}
-Advisory Membership is open to all members of the Rochester Institute of Technology professional, academic, or administrative staff.
+Advisory Membership is open to all current members of the Rochester Institute of Technology professional, academic, or administrative staff.
```

Comment:
I don't think current is necessary here. You either are a member of the staff or you aren't.